docs/src/language_servers.md

```diff
@@ -627,6 +627,23 @@ Follow installation instructions on [LSP-metals](https://github.com/scalameta/me
     }
     ```
 
+### Solidity
+
+1. Install [github:NomicFoundation/hardhat-vscode](https://github.com/NomicFoundation/hardhat-vscode/tree/development/server) (follow instructions in the repo):
+
+2. Open `Preferences > Package Settings > LSP > Settings` and add the `"solidity"` client configuration to the `"clients"`:
+
+    ```jsonc
+    {
+        "clients": {
+            "solidity": {
+                "command": ["/PATH/TO/node", "PATH/TO/nomicfoundation-solidity-language-server", "--stdio"],
```

Comment:
Haven't checked but should be enough with something like:

```suggestion
                "command": ["nomicfoundation-solidity-language-server", "--stdio"],
```

Reply:
I've started with just `nomicfoundation-solidity-language-server`, but it didn't work for me, it wasn't able to locate `node` for some reason 🤔 Will take another look if there were any misconfig on my side

Reply:
If node is not in the path then I would consider it a separate issue that is explained at https://lsp.sublimetext.io/troubleshooting/#updating-the-path-used-by-lsp-servers